zebra-chain/src/orchard/tests/vectors/key_components.rs

```diff
@@ -4,7 +4,6 @@
 //! Produced by <https://github.com/zcash-hackworks/zcash-test-vectors/blob/ec5fe3abef5219d0f8c9edbc93bb4038f1729dfe/orchard_key_components.py>
 
 #![allow(dead_code)]
```

Comment:
Did you mean to delete `#![allow(dead_code)]` instead?

Reply:
Not really, this is needed but only for the `TestVector` struct, not the whole file.